specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2025-09-01/workspaceRP.json

```diff
@@ -2733,6 +2733,16 @@
       "description": "Managed Network settings for a machine learning workspace.",
       "type": "object",
       "properties": {
+        "enableFirewallLog": {
+          "description": "A flag to indicate if monitoring needs to be enabled for the managed network firewall.",
+          "type": "boolean",
+          "default": false
+        },
```

Comment:
this is ok by me but remember the usual recommendation: enums are more extensible than bools...

e.g.

`"firewallLogging": "Enabled" | "Disabled"`

could be extended with future option values

whereas 

`"enableFirewallLog": true | false`

cannot.